### DISCUSSION TITLE
Change notice versus Contract Modification Notice

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#553] [Q&A] [answered]
Hi lab , this time I do not ask a 'technical' question ..
About Change Notice it is clear .. for any notice there is another section , for any notice CAN included.
I guess that I can change all values of the 'old notice'  ( except the 'famous' 10 fields ) and the new notice will replace the old one.
It is right ?
With the Contract Modification Notice we do not start from the 'old' notice that is the notice that we want modify , there are all usual sections, like Contracting party and service provider, Procedure, Lot, etc. There is Modification Section,  that 'link' to the old notice, but what about the other sections ?
I could do a change notice of a CAN notice, what can do 'more' on the Contract Modification Notice ?
Thanks a lot, and Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
All sections related to the contract being modified are necessary, others shouldn't be there. We currently do not have rules in place and this will be done in a future release.

**Comment by rengit62** [comment]
Hi, perhaps my question was not clear.
In doc it is written : 
**A contract modification follows the same structure as a change, it however appears during contract execution and applies to a CAN only."**

What does it mean "it however appears during contract execution" ?  That is , when the user could fill this notice ( from 38 to 40 ), rather than a 'common' change notice ? If course I am talking about **only** the CAN notice ( from 25 to 37 )

Thanks and Regards

**Reply by YvesJo** [reply]
Change is a corrigenda while Modification is about an active contract.

**Reply by rengit62** [reply]
Ok , last question :
By eNotices2 is it possible to see these notices ( I mean 38, 39 and 40 )

**Reply by YvesJo** [reply]
Hi,
for non technical questions please get in touch with the helpdesk at ted@publications.europa.eu. These forms are there and there is the need to have a published Result notice though.
